azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/action/LifecycleAction.kt

```diff
@@ -4,9 +4,9 @@
 package com.azure.android.communication.ui.calling.redux.action
 
 internal sealed class LifecycleAction : Action {
-    class EnterForegroundTriggered : LifecycleAction()
-    class EnterBackgroundTriggered : LifecycleAction()
+    object EnterForegroundTriggered : LifecycleAction()
```

Comment:
Do we need triggered/succeeded?

We are dispatching after this has happened, it's already "succeeded"

I think EnterForeground, EnterBackground is probably enough.

Reply:
fixed